src/projections/robin.cpp

```diff
@@ -142,6 +142,9 @@ static PJ_LP robin_s_inverse (PJ_XY xy, PJ *P) {           /* Spheroidal, invers
         lp.phi = (5 * i + t) * DEG_TO_RAD;
         if (xy.y < 0.) lp.phi = -lp.phi;
         lp.lam /= V(X[i], t);
+        if( fabs(lp.lam) > M_PI ) {
+            lp.lam = lp.phi = HUGE_VAL;
```

Comment:
I think it would be nice to set an error here. Something like:
```suggestion
            proj_errno_set(P, PJD_ERR_LAT_OR_LON_EXCEED_LIMIT);
            lp = proj_coord_error().lp;
```

There may be a better error code to set but if I understand the problem correctly this one fits the bill.

Reply:
I'm fine with this